message MSG_CLIENT_DM_ROOT_DESTORY
{
    optional string token = 1;
    optional string userName = 2;
}

message MSG_DM_CLIENT_ROOT_DESTORY_ACK
{
    optional int32 statusCode = 1;
}